core/templates/dev/head/css/oppia.css

```diff
@@ -753,6 +817,118 @@ textarea {
   border: 0;
   margin-bottom: 0;
   width: 100%;
+  padding: 0;
+}
+.navbar-form {
+  padding-left: 15px;
+}
+
+.nav {
+  padding-left: 0;
+  margin-bottom: 0;
+  list-style: none;
+}
+.nav > li > a {
+  position: relative;
+  display: block;
+  padding: 10px 15px;
+}
+.nav > li {
+  position: relative;
+  display: block;
+}
+
+.open > a {
+  outline: 0;
+}
+.open > .dropdown-menu {
+  display: block;
+}
+.dropdown-menu {
+  padding: 5px 0;
+  margin: 2px 0 0;
+  max-width: 220px;
+  min-width: 160px;
+  font-size: 14px;
+  -webkit-background-clip: padding-box;
+  background-clip: padding-box;
+  -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, .175);
+  box-shadow: 0 6px 12px rgba(0, 0, 0, .175);
+}
+.float-right > .dropdown-menu {
+  right: 0;
+  left: auto;
+}
+.dropdown-item.active, .dropdown-item:active {
+  background-color: transparent;
+}
+.oppia-navbar-profile-picture-container .caret {
+  margin: 4px;
+  margin-left: 10px;
+}
+.dropup .caret,
+.navbar-fixed-bottom .dropdown .caret {
+  content: "";
+  border-top: 0;
+  border-bottom: 4px solid;
+}
+.dropup .dropdown-menu,
+.navbar-fixed-bottom .dropdown .dropdown-menu {
+  top: auto;
+  bottom: 100%;
+  margin-bottom: 2px;
+}
+.nav-item {
+  position: relative;
+  display: block;
+}
+.nav-link {
+  display: block;
+  font-size: 14px;
+  position: relative;
+}
+.nav-link.dropdown-item {
+padding: 3px 20px 2px 20px;
+}
+.navbar-nav.oppia-navbar-nav .nav-link {
+    color: #fff;
```

Comment:
Indent by 2 not 4.

In general I should not need to comment on style stuff like alphabetization and indentation. Please check that everything is done correctly before you ask for a review.

Reply:
Done missed this. Thanks!